src/pages/challenges.tsx

```diff
@@ -27,7 +27,8 @@ const Challenges: FC = () => {
     error,
     isLoading,
     callAPI } = useFetch(CHALLENGES_URL, {}, false);
-  const { isLoggedIn, isLoading: isAuthenticating } = useAuthenticated();
+  const { state } = useAppContext();
+  const { isLoading: isAuthenticating, isLoggedIn } = state;
 
   useEffect(() => {
```

Comment:
we won't need this effect if we take the userData from the context itself right?